py/server/deephaven/_udf.py

```diff
@@ -211,6 +216,9 @@ def prepare_auto_arg_conv(self, encoded_arg_types: str) -> bool:
         for arg_type_str, param in zip(arg_type_strs, self.params):
             param.setup_arg_converter(arg_type_str)
 
+        if isinstance(self.fn, (numba.np.ufunc.gufunc.GUFunc, numba.np.ufunc.dufunc.DUFunc)) or isinstance(self.fn, numpy.ufunc):
+            warnings.filterwarnings("default", category=UserWarning)
+
```

Comment:
I'm not sure what is supposed to be happening here.
1. This code is unreachable
2. I'm not sure what the warnings filter is doing

Reply:
Should have cleaned up the changes a bit more